UPGRADE-7.0.md

```diff
@@ -136,6 +156,18 @@ Serializer
  * First argument of `AttributeMetadata::setSerializedName()` is now required
  * Add argument `$context` to `NormalizerInterface::supportsNormalization()` and `DenormalizerInterface::supportsDenormalization()`
 
+Translation
+-----------
+
+ * Remove `PhpStringTokenParser`
+ * Remove `PhpExtractor` in favor of `PhpAstExtractor`
+
+TwigBundle
+----------
+
+ * Remove option `twig.autoescape`, use `twig.autoescape_service[_method]` instead
```

Comment:
I suggest adding a bit more info (or a code example) here as well. Even with #46052 , I'm not 100% what I should do if one of my apps is using the removed `twig.autoescape` option.

Reply:
added